### DISCUSSION TITLE
Bump to support Vite 4

### DISCUSSION THREAD

**Discussion by craigrileyuk** [kucrut/vite-for-wp#31] [Ideas]
First of all, thanks for creating this package. It's been extremely useful.

Recent Vite 4 was released with its major change being a bump in support to Rollup 3. 

Would it be possible to alter the dependencies for this package to allow its installation alongside Vite 4, assuming no conflicts.

Thanks.

**Comment by kucrut** [comment]
Done. Version 0.5.2 depends on Vite 4.0.1.

**Reply by craigrileyuk** [reply]
Awesome. Thanks for doing that!
